app/browser/api/ledgerNotifications.js

```diff
@@ -51,6 +59,10 @@ const shouldShowNotificationAddFunds = () => {
   const nextTime = getSetting(settings.PAYMENTS_NOTIFICATION_ADD_FUNDS_TIMESTAMP)
   return !nextTime || (new Date().getTime() > nextTime)
 }
+const shouldShowNotificationBackupKeys = (state) => {
```

Comment:
this seems redundant since https://github.com/brave/browser-laptop/pull/13504/files#diff-3920198635a4b5a4d7e27313620f971dR240 already checks if the difference between `now` and `bootStamp` is greater than the requisite time interval

Reply:
That may be the case. The number of refactors could have made it obsolete. I double check my numbers to make sure. Thanks